jbpm-test-coverage/src/test/java/org/jbpm/test/functional/log/ProcessInstanceLogCleanTest.java

```diff
@@ -385,9 +380,10 @@ private void abortProcess(KieSession kieSession, long pid) {
         }
     }
 
-    private List<ProcessInstance> startProcess(KieSession kieSession, String processId, int count) {
+    private List<ProcessInstance> startProcess(KieSession kieSession, String processId, int count) throws InterruptedException {
         List<ProcessInstance> piList = new ArrayList<ProcessInstance>();
         for (int i = 0; i < count; i++) {
+            TimeUnit.MILLISECONDS.sleep(1000);
```

Comment:
Reduce sleep time (probably a few milliseconds is enough)

Reply:
agree